src/java.base/share/classes/java/lang/reflect/code/interpreter/Interpreter.java

```diff
@@ -328,6 +328,85 @@ Object invoke(MethodHandles.Lookup l, T invokableOp,
         }
     }
 
+    static <T extends Op & Op.Invokable>
```

Comment:
Can we refactor to also share this code in the `invoke` method rather than duplicate?

Suggest renaming from `invokeBody` to `interpretBody`.